### PR TITLE
#563 fix map upload bad request

### PR DESCRIPTION
**Pull request by Geosearchef**

### CODE REVIEW

File: src/main/java/com/faforever/client/api/FafApiAccessorImpl.java [hunk 1]
```diff
@@ -257,6 +257,7 @@ public void uploadMod(Path file, ByteCountListener listener) {
   @Override
   public void uploadMap(Path file, boolean isRanked, ByteCountListener listener) {
     MultiValueMap<String, Object> multipartContent = createFileMultipart(file, listener);
+    multipartContent.add("metadata", "{\"isRanked\":\"" + isRanked + "\"}");
```
Comment (Geosearchef) [inline src/main/java/com/faforever/client/api/FafApiAccessorImpl.java]: Maybe a more elegant way to do this? Like using an object mapper as in the API (can't be autowired as it's not a bean in the client). Creating an extra class for this is a bit overkill.

Comment (Brutus5000) [inline src/main/java/com/faforever/client/api/FafApiAccessorImpl.java]: I'm a bit confused. That doesn't look like a solution to any of the problems I have encountered so far. Can you give specific info which error message you intend to fix here?

Comment (Geosearchef) [inline src/main/java/com/faforever/client/api/FafApiAccessorImpl.java]: The 400 bad request issue #563 . With this line added I'm able to upload maps perfectly fine.

**Comment by micheljung** [comment]
Also, please Write "Fix #563" into the commit message, so that the related issue gets closed automatically as soon as this PR is merged

**Pull request merged**